### ISSUE TITLE
Reaction selector does not support keyboard navigation

### ISSUE THREAD

**Issue by hifi** [cinnyapp/cinny#58]
#### Describe the bug
Reaction selector requires mouse to select an emoji from the list. 

#### Expected behavior
When you have filtered emojis with a text string you should be able to either use the arrow keys or tab key to select the wanted reaction and hit enter to apply your selection.

If it's hard to implement arrow keys as it conflicts with the text input just having tab and shift-tab do the expected thing (go through the list and go back to input beyond the first one) it would already be a great improvement.

#### Additional context
Standard UX for reaction selectors. Slack and Discord both support full arrow key motions to browse the suggestions.

**Comment by jasikpark** [comment]
Additionally, to be more accessible to keyboard users, the modal should focus on display! 🙏 thanks for this amazing client btw!
